### ISSUE TITLE
Better knocking sound

### ISSUE THREAD

**Issue by AuracleTech** [MeowMC/Door-Knocker#1]
This is the sounds I used for my version that I commented on the plugin.

You can test them if you want, they are royalty free, high quality, one single knocking per click allowing you to manually knock on the door increasing the immersion!

[Sounds.zip](https://github.com/MeowMC/Door-Knocker/files/5832383/Sounds.zip)

**Comment by froyo4u** [comment]
Someone actually made a PR for splitting the sounds! Closing issue ;)

**Comment by MeowMC** [comment]
Wait I can't close it tf

**Comment by MeowMC** [comment]
Nice job mobile github

**Comment by froyo4u** [comment]
Closed, added with a PR

**Comment by AuracleTech** [comment]
I just listened to the sound he used and the quality is terrible, also there's background noise and it isn't cut properly.
Mine was clean, zero delay and royalty free...

Sad you didn't accept them...

**Comment by froyo4u** [comment]
I didn't know

**Comment by ZandercraftGames** [comment]
I will look into this.

**Comment by ZandercraftGames** [comment]
@AuracleTech Hey again. So I noticed that the sound you provided is a single knock, but it does not include other variants. Do you maybe have a grouping of variants, as all knocks don't generally sound the same in real life? Thanks! :D

**Comment by AuracleTech** [comment]
Too bad I guess, but it was still way better.

**Comment by froyo4u** [comment]
Reopening, I'll take the sound and make copies and just pitch adjust the copies to have variants of the sound, cuz I feel bad

**Comment by AuracleTech** [comment]
I don't force you to I don't want you to take pity on me neither I'm just saying mine is better quality and better clipped, but I mean, it's your mod your choice

**Comment by froyo4u** [comment]
> I don't force you to I don't want you to take pity on me neither I'm just saying mine is better quality and better clipped, but I mean, it's your mod your choice

I know, I just feel yours is better

**Comment by ZandercraftGames** [comment]
> I don't force you to I don't want you to take pity on me neither I'm just saying mine is better quality and better clipped, but I mean, it's your mod your choice

Yes, my apologies for making you feel I was rejecting your issue. I was simply wondering if you had multiple variants of like pitch or woodiness so that it could sound more natural when knocking multiple times in a row. 😄

**Comment by AuracleTech** [comment]
>  I was simply wondering if you had multiple variants of like pitch or woodiness so that it could sound more natural when knocking multiple times in a row. 😄
>
To do this for games and other purpose I use quick pitch change in audacity like 1-2% difference, I'll send here a zip with new variants for fit his new sound system

**Comment by ZandercraftGames** [comment]
> >  I was simply wondering if you had multiple variants of like pitch or woodiness so that it could sound more natural when knocking multiple times in a row. 😄
> >
> To do this for games and other purpose I use quick pitch change in audacity like 1-2% difference, I'll send here a zip with new variants for fit his new sound system

Thank you so much! 😁
I really appreciate it! 😄

**Comment by AuracleTech** [comment]
Best of both worlds, new system with new sounds.
Hope you like it!
Preview of [Before](https://voca.ro/128BL0P1n3SS) and [After](https://voca.ro/1av5ArlVRvHG)
[NewSounds.zip](https://github.com/MeowMC/Door-Knocker/files/6138708/NewSounds.zip)

**Comment by ZandercraftGames** [comment]
I don't know about Meow's thoughts on this, but I think it sounds good. I will definitely implement this into the Forge version of Door Knocker. If Meow approves, I will also PR it into the Fabric version.
